digitalpublicgoods/safe-you.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "Safe YOU",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.linkedin.com/showcase/safe-you"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No"
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/safeyou-space/safeyou/blob/main/README.md"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "Extraction of non PII is made through a manual mechanism from database. It still remains a restricted information and can be extracted only by authorized personnel."
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "The Project is currently active in Armenia and Georgia. Our Policies comply with Domestic information Security laws, both in Armenia and Georgia. More specifically, this refers to “RA Civil Code” adopted on 5 May 1998, “RA Law on protection of personal data” adopted on 18 May 2015. For Georgia: \"Law of Georgia on Information Security\" adopted on 01/07/2012. We are currently reviewing our policies to comply with GDPR."
+    ],
+    "adherenceSteps": [
+      "https://docs.google.com/document/d/1OMw2OnSP-_SaXWai7kZzPeqd71RrQo2VEP_T2NUxazo/edit?usp=sharing https://docs.google.com/document/d/1Px4lfHdbUUzgH5tEW2AyQOkvorPPOau7NglPfZdBEdA/edit?usp=sharing"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Unknown",
```

Comment:
This should not be `unknown`, they need to support some standards.

Reply:
Resolved in commit 1fac266